tests/shared/UserLogonRequestTestDefs.js

```diff
@@ -12,30 +12,151 @@
 
 "use strict";
 
-var fluid = require("infusion"),
+var fluid  = require("infusion"),
     jqUnit = fluid.require("node-jqunit", require, "jqUnit"),
-    gpii = fluid.registerNamespace("gpii");
+    gpii   = fluid.registerNamespace("gpii");
 
 fluid.require("%gpii-universal");
 
-fluid.require("%gpii-universal/gpii/node_modules/testing/src/PromiseUtils.js");
-
 gpii.loadTestingSupport();
 
 fluid.registerNamespace("gpii.tests.userLogonRequest");
 
+gpii.tests.userLogonRequest.gpiiKey = "adjustCursor";
+gpii.tests.userLogonRequest.anotherGpiiKey = "sammy";
+
 fluid.defaults("gpii.tests.userLogonRequest.testCaseHolder", {
-    gradeNames: [ "gpii.test.common.lifecycleManagerReceiver", "gpii.test.common.testCaseHolder" ],
+    gradeNames: ["gpii.test.common.lifecycleManagerReceiver", "gpii.test.testCaseHolder"],
     events: {
-        debounceTimeoutComplete: null,
-        onResponse: null,
-        onError: null
+        onNoUserLoggedIn: null,
+        debounceTimeoutComplete: null
+    },
+    expectedModelChanges: {
+        proximityTriggeredLogon: [
```

Comment:
I think this model makes a lot more sense

Reply:
I'm glad, I was shooting for something clearer and reliable.